flow/project.py

```diff
@@ -557,6 +557,9 @@ def __init__(self, condition, tag=None):
                 super(pre, self).__init__(condition, tag)
 
             def __call__(self, func):
+                for (name, fun) in self._parent_class._collect_operations():
```

Comment:
@mikemhenry @ac-optimus The underscore at the beginning indicates a private/internal symbol. To re-use a symbol name, you want `func_` instead of `_func`. See PEP 8: https://www.python.org/dev/peps/pep-0008/#descriptive-naming-styles

Reply:
ya you are correct, I have changed it to func_ @bdice @mikemhenry!